spec/views/spotlight/search_configurations/_search_fields.html.erb_spec.rb

```diff
@@ -27,8 +27,7 @@
   end
 
   it 'has a checkbox to enable or disable fielded search' do
-    expect(rendered).to have_selector 'input[data-behavior="enable-feature"][data-target="#search_fields"], ' \
-                                      'input[data-behavior="enable-feature"][data-bs-target="#search_fields"]'
+    expect(rendered).to have_selector 'input[data-behavior="enable-feature"][data-target="#search_fields"]'
```

Comment:
these are not bootstrap targets so we should just use `data-target` not `data-bs-target`